apps/judicial-system/web/src/routes/Shared/Cases/ActiveCases.tsx

```diff
@@ -360,59 +345,41 @@ const ActiveCases: React.FC<React.PropsWithChildren<Props>> = (props) => {
                         c.state === CaseState.DRAFT ||
                         c.state === CaseState.SUBMITTED ||
                         c.state === CaseState.RECEIVED) && (
-                        <motion.button
-                          key={`${c.id}-delete`}
-                          initial={{ opacity: 0 }}
-                          animate={{ opacity: 1 }}
-                          exit={{ opacity: 0 }}
-                          data-testid="deleteCase"
-                          aria-label="Viltu afturkalla kröfu?"
-                          className={cn(
-                            styles.deleteButton,
-                            styles.deleteButtonWrapper,
-                          )}
-                          onClick={async (evt) => {
-                            evt.stopPropagation()
-
-                            await new Promise((resolve) => {
-                              setRequestToRemoveIndex(
-                                requestToRemoveIndex === i ? undefined : i,
-                              )
-
-                              resolve(true)
-                            })
-
-                            await controls.start('isDeleting')
-                          }}
-                        >
-                          <Icon icon="close" color="blue400" />
-                        </motion.button>
+                        <ContextMenu
+                          menuLabel={`Valmynd fyrir mál ${c.courtCaseNumber}`}
+                          items={[
+                            {
+                              title: formatMessage(m.contextMenu.openCase),
+                              onClick: () => handleOpenCase(c.id, true),
+                              icon: 'open',
+                            },
+                            {
+                              title: formatMessage(m.contextMenu.deleteCase),
+                              onClick: async () => {
```

Comment:
Just wondering, should we be worried about the possibility of someone accidentally deleting a case when trying to open in a new tab since the 2 options are right next to each other? Is there a need for some kind of confirmation before the deletion happens?

Reply:
Awesome point! Thanks.. Resolved IRL..